server/config-example.js

```diff
@@ -34,4 +34,4 @@ config.couchDbURL = config.getProtocol(config.couchDbUseSsl) + config.couchDbSer
 config.couchAuthDbURL = config.getProtocol(config.couchDbUseSsl) + config.couchCredentials() + config.couchDbServer + ':' + config.couchDbPort;
 // config.searchURL = 'http://localhost:9200'; ELASTIC SEARCH URL (OPTIONAL)
 config.serverInfo = 'Development Ember CLI server';
-module.exports = config;
+module.exports = config;
```

Comment:
Also, I realized that this size parameter never makes it to the call to elasticsearch.